scripts/etc/init.d/rcS

```diff
@@ -838,29 +838,43 @@ echo "OK"
 # networking
 echo -n "  Configuring network settings... "
 touch /rootfs/etc/network/interfaces || fail
-{
-    # wheezy seems to add the lo interface, so first check for it
-    if ! grep -q "auto lo" /rootfs/etc/network/interfaces; then
-        echo "auto lo"
-        echo "iface lo inet loopback"
-        echo ""
-    fi
+# lo interface may already be there, so first check for it
+if ! grep -q "auto lo" /rootfs/etc/network/interfaces; then
+    echo "auto lo"                      >> /rootfs/etc/network/interfaces
+    echo "iface lo inet loopback"       >> /rootfs/etc/network/interfaces
+    echo ""                             >> /rootfs/etc/network/interfaces
```

Comment:
when echo-ing an empty line, no quotes are needed (or desired, as it makes it clear(er) that the intention was an empty line)...

that said, I don't think an empty line is needed here at all... if anywhere, add it before adding additional "entries"